staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/errors/statuserror_test.go

```diff
@@ -152,3 +155,34 @@ func TestToStatusErr(t *testing.T) {
 		}
 	}
 }
+
+func TestNewDryRunUnsupportedErr(t *testing.T) {
+	type args struct {
+		webhookName string
+	}
+	tests := []struct {
```

Comment:
If you have only one test, you don't specifically need to turn this into a table-test, just compare the thing you want to compare :-)